.github/workflows/scripts/run-integration-tests-group.sh

```diff
@@ -36,11 +42,20 @@ if [[ -z "$TOTAL" ]]; then
     exit 1
 fi
 
+export IMAGE_TAG_RACE=${IMAGE_TAG_RACE:-$(make image-tag-race)}
+export MIMIR_IMAGE=${MIMIR_IMAGE:-"grafana/mimir:$TAGPREFIX$IMAGE_TAG_RACE"}
+export IMAGE_TAG=${IMAGE_TAG:-$(make image-tag)}
+export MIMIRTOOL_IMAGE=${MIMIRTOOL_IMAGE:-"grafana/mimirtool:$IMAGE_TAG"}
+export MIMIR_CHECKOUT_DIR=${MIMIR_CHECKOUT_DIR:-"/go/src/github.com/grafana/mimir"}
+
```

Comment:
Calling `make` from the script relies on the script being called from mimir root dir. Other parts of this script don't make such assumption. We could compute `MAKEFILE_PATH` (using `$SCRIPT_DIR`) and pass `-f` option to `make` explicitly to make sure that correct `Makefile` is used.